mozilla-release/toolkit/mozapps/installer/upload-files.mk

```diff
@@ -231,25 +232,33 @@ endif
 #For each of the main, tests, sdk rpms we want to make sure that
 #if they exist that they are in objdir/dist/ and that they get
 #uploaded and that they are beside the other build artifacts
-MAIN_RPM= $(MOZ_APP_NAME)-$(MOZ_NUMERIC_APP_VERSION)-$(MOZ_RPM_RELEASE).$(BUILDID).$(TARGET_CPU)$(PKG_SUFFIX)
-UPLOAD_EXTRA_FILES += $(MAIN_RPM)
+MAIN_RPM= $(MOZ_APP_NAME)-$(MOZ_NUMERIC_APP_VERSION)-$(MOZ_RPM_RELEASE).$(BUILDID).$(TARGET_CPU)$(RPM_PKG_SUFFIX)
+MAIN_DEB= $(MOZ_APP_NAME)-$(MOZ_NUMERIC_APP_VERSION)-$(MOZ_RPM_RELEASE).$(BUILDID).$(TARGET_CPU).deb
+# Create deb package
 RPM_CMD += && mv $(TARGET_CPU)/$(MAIN_RPM) $(_ABS_DIST)/
-
-ifdef ENABLE_TESTS
-TESTS_RPM=$(MOZ_APP_NAME)-tests-$(MOZ_NUMERIC_APP_VERSION)-$(MOZ_RPM_RELEASE).$(BUILDID).$(TARGET_CPU)$(PKG_SUFFIX)
-UPLOAD_EXTRA_FILES += $(TESTS_RPM)
-RPM_CMD += && mv $(TARGET_CPU)/$(TESTS_RPM) $(_ABS_DIST)/
-endif
-
-ifdef INSTALL_SDK
-SDK_RPM=$(MOZ_APP_NAME)-devel-$(MOZ_NUMERIC_APP_VERSION)-$(MOZ_RPM_RELEASE).$(BUILDID).$(TARGET_CPU)$(PKG_SUFFIX)
-UPLOAD_EXTRA_FILES += $(SDK_RPM)
-RPM_CMD += && mv $(TARGET_CPU)/$(SDK_RPM) $(_ABS_DIST)/
-endif
-
-INNER_MAKE_PACKAGE = $(RPM_CMD)
+RPM_CMD += && fakeroot alien $(MAIN_RPM) --scripts
+# Rename newly created deb (alien does not provide such option)
+RPM_CMD += && mv *.deb $(MAIN_DEB)
+UPLOAD_EXTRA_FILES += $(MAIN_RPM)
+UPLOAD_EXTRA_FILES += $(MAIN_DEB)
+
+#ifdef ENABLE_TESTS
+#TESTS_RPM=$(MOZ_APP_NAME)-tests-$(MOZ_NUMERIC_APP_VERSION)-$(MOZ_RPM_RELEASE).$(BUILDID).$(TARGET_CPU)$(RPM_PKG_SUFFIX)
+#UPLOAD_EXTRA_FILES += $(TESTS_RPM)
+#RPM_CMD += && mv $(TARGET_CPU)/$(TESTS_RPM) $(_ABS_DIST)/
+#endif
+
+#ifdef INSTALL_SDK
+#SDK_RPM=$(MOZ_APP_NAME)-devel-$(MOZ_NUMERIC_APP_VERSION)-$(MOZ_RPM_RELEASE).$(BUILDID).$(TARGET_CPU)$(RPM_PKG_SUFFIX)
+#UPLOAD_EXTRA_FILES += $(SDK_RPM)
+#RPM_CMD += && mv $(TARGET_CPU)/$(SDK_RPM) $(_ABS_DIST)/
+#endif
```

Comment:
Please use some form of multiline commenting instead of changing every line.
If normal `#if 0` won't make it, then
```
#ifeq (1, 0)
... 
#endif
``` 
may work.
And please put a comment on why exactly you disable this piece of code.